### PR TITLE
AF-2140: Fix restoring a custom editor

### PR DESCRIPTION
**Pull request by tiagobento**
Also on this PR:

- List of Known Issues on README.md
- IT for restoring a custom editor

### CODE REVIEW

**Comment by domhanak** [comment]
@tiagobento We should also report these issues in JIRA or `Issues` section here on github.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ yarn run build:fast # skips lint and unit tests
 
 > Final artifacts will be on `packages/*/dist` directories.
 
+Known issues
+--------------------
+- Flicker when opening or restoring DMN/BPMN editors.
+- Unable to detect 'dirty' state on DMN/BPMN editors. This means that when closing an editor with changes, we can't show a warning popup to remind the user to save it before closing.
+- Copy/paste diagram parts between DMN/BPMN editors.
```
Comment (domhanak) [inline README.md]: I suggest we either put these in Issues or in JIRA. It will allow discussions, increase visibility and allow better planning.

**Review by domhanak — CHANGES_REQUESTED** [review]
please do report the issues in KOGITO jira and link them to readme. Let's use `kogito-tooling` label for our issues. otherwise ok.

**Comment by tiagobento** [comment]
@domhanak I think we can do that, but I think having the known issues on the README is also valuable for people who don't know about our JIRA.

**Comment by domhanak** [comment]
@tiagobento Yea I agree, we can link JIRA there :)

**Comment by tiagobento** [comment]
@domhanak Done. Are we good to merge?

**Review by domhanak — APPROVED** [review]

**Pull request merged**